src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala

```diff
@@ -328,8 +360,8 @@ class FastSync(
             handleRewind(header, peer, syncConfig.fastSyncBlockValidationN)
           case HeadersProcessingFinished =>
             processSyncing()
-          case ImportedTargetBlock  =>
-            updateTargetBlock(ImportedLastBlock)
+          case ImportedTargetBlock =>
```

Comment:
should be still called `ImportedTargetBlock`? ...same for `waitingForTargetBlockUpdate`